docs/src/Root.js

```diff
@@ -34,6 +34,7 @@ const Root = props => {
     <>
       <Helmet>
         <title>{title}</title>
+        <link rel="shortcut icon" type="image/png" href="/logo.png"/>
```

Comment:
Logo in this case is 512 size, maybe a smaller 64x64 file for favicon.png or favigon.ico would be a better usage than logo.